src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -167,6 +171,14 @@ class DockerCreateContainer extends AbstractDockerRemoteApiTask {
         exposedPorts << new ExposedPort(internetProtocol, ports)
     }
 
+    void setRestartPolicy(String name, int maximumRetryCount) {
+        restartPolicy = "${name}:${maximumRetryCount}"
+    }
+
+    void setRestartPolicy(String serialized) {
```

Comment:
Lets remove this method. If folks want to specify the restartPolicy directly they can do so (restartPolicy = "random:123"). No need to have an extra method.